### PR TITLE
Suggestions on the initial toolbox PR

### PR DESCRIPTION
**Pull request by jkgoodrich**

### CODE REVIEW

**Review by KoalaQin — CHANGES_REQUESTED** [review]
I'm sending these over for now because I need 1 of them for testing.

**Review by KoalaQin — CHANGES_REQUESTED** [review]
Thank you for making this PR, your new structure of code makes more sense! 
I think we can merge yours after some changes and I could work on the to-dos.

File: gnomad_toolbox/filtering/variant.py [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+    )
+
+    # Filter to the variant of interest.
+    ht = ht.filter(ht.alleles == variant.alleles)
+
+    # Check if the variant exists.
+    if ht.count() == 0:
+        hl.utils.warning(
+            f"No variant found at {variant.locus} with alleles {variant.alleles}"
+        )
```
[the remark this answers is not in the document]
Comment (jkgoodrich) [reply gnomad_toolbox/filtering/variant.py:76]: really? I got the warning

File: README.md [hunk 2]
```diff
@@ -1,2 +1,33 @@
...
+│
+├── analysis/
+│   ├── __init__.py
+│   ├── general.py       # General analysis functions, such as summarizing variant statistics.
+│
+├── notebooks/
+│   ├── intro_to_release_data.ipynb    # Jupyter notebook introducing the loading of gnomAD release data.
+```
+
+# TODO: Add fully detailed info about how to install and open the notebooks.
```
[the remark this answers is not in the document]
Comment (jkgoodrich) [reply README.md:27]: Sounds good, but don't add anything yet. First step is to get a bunch of tickets in for todo's and we can decide who will tackle what

File: gnomad_toolbox/filtering/variant.py [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+    #                exon.stop + 75,
+    #                reference_genome=build,
+    #                includes_end=True,
+    #            )
+    #        ),
+    #        gene_ht.exons.filter(lambda exon: exon.feature_type == "CDS"),
+    #    )
+    # )
+
+    # TODO: Consider this alternative approach to get the intervals from gencode. That
```
[the remark this answers is not in the document]
Comment (jkgoodrich) [reply gnomad_toolbox/filtering/variant.py:168]: OK, lets go with the faster way and a filter so we get the same numbers as the browser would get. For now lets just add a TODO and a ticket to do it

File: gnomad_toolbox/filtering/variant.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (KoalaQin) [reply gnomad_toolbox/filtering/variant.py:76]: Yeah, it's working, but it was printed after the init cell. Screenshot 2024-12-18 at 1 17 25 PM

[the remark this answers is not in the document]
Comment (KoalaQin) [reply gnomad_toolbox/filtering/variant.py:76]: it's just a hail/notebook thing.

**Review by KoalaQin — APPROVED** [review]
LGTM!

**Pull request merged**